delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/ResourcesRoutes.scala

```diff
@@ -171,6 +171,22 @@ final class ResourcesRoutes(
                           }
                         )
                       },
+                      (pathPrefix("updateSchema") & put & pathEndOrSingleSlash) {
```

Comment:
`update-schema` ?
Not sure if we have already occurrences of path where we have this case